smartDevices/src/main/java/sci/homework/oop2/SmartHome.java

```diff
@@ -0,0 +1,40 @@
+package sci.homework.oop2;
+
+/**
+ * clasa abastracta care defineste o casa Smart
+ * by defaul toate obiecetele din clasa abstracta SmartHome au un consum de 0.1 pentru modul standby unde asteapta comezi
+ */
+public abstract class SmartHome implements SmartDevice {
```

Comment:
This class looks more like an AbstractSmartDevice (i.e abstract implementation of the SmartDevice interface)